kong/plugins/jwt/access.lua

```diff
@@ -65,10 +65,15 @@ function _M.execute(conf)
 
   local claims = jwt.claims
 
-  local jwt_secret_key = claims.iss
+  local jwt_secret_key_field = "iss"
+  if conf.secret_key_field then
+    jwt_secret_key_field = conf.secret_key_field
+  end
```

Comment:
This is unnecessary. The schema validation guarantees that conf.secret_key_field will be set to `iss` if no other value is provided. You can directly use it through the code.